protos/feast/types/Field.proto

```diff
@@ -27,4 +27,7 @@ option go_package = "github.com/feast-dev/feast/go/protos/feast/types";
 message Field {
   string name = 1;
   feast.types.ValueType.Enum value = 2;
+
+  // User defined metadata
+  map<string,string> tags = 3;
```

Comment:
I don't think this change is necessary rn; we've been using `FeatureSpecV2` as the underlying proto structure

that being said, if you want we can migrate from using `FeatureSpecV2` to using `Field`

Reply:
I will migrate, I think that makes more sense.